### ISSUE TITLE
doCheckHouses.lua

### ISSUE THREAD

**Issue by Lycanzito** [opentibiabr/otservbr-global#883]
We'have a problem with : doCheckHouses.lua.

If  the player loses his house due to the fact that he stays X days without logging in, the items do not go to his depot, so that the next person simply buys the house and takes his items.

![image](https://user-images.githubusercontent.com/52710697/75311326-2a098780-5835-11ea-9f1e-45eb1c3f4963.png)

```
local function doCheckHouses()

    local dias = 14
    local tempo = os.time() - (dias * 24 * 60 * 60)
    local registros = db.storeQuery("SELECT `houses`.`owner`, `houses`.`id` FROM `houses`,`players` WHERE `houses`.`owner` != 0 AND `houses`.`owner` = `players`.`id` AND `players`.`lastlogin` <= " .. tempo .. ";")

    if registros ~= false then

        local count = 0

        print('house leave code')

        repeat
            count = count + 1

            local owner = result.getNumber(registros, "owner")
            local houseId = result.getNumber(registros, "id")
            local house = House(houseId)

            if house and (owner > 0) then
                print(house:getName())
                house:setOwnerGuid(0)
            end

        until not result.next(registros)

        print('house leave house count:' .. count)

        result.free(registros)
    end
end

function onStartup()
    addEvent(doCheckHouses, 60 * 1000)

    return true
end
```

**Comment by Lycanzito** [comment]
Talvez
tenha relação com os carpets, que eu reportei , mas não consigo entender, o pq teria relação com as outras casas, pq ele limpa toooodos os carpetes, de tooodas as casas.

**Comment by dudantas** [comment]
Solved on the https://github.com/opentibiabr/otservbr-global/commit/67433c0c1b4f62f0a87ca4d99723ad8d49747fa2

Thanks.

**Comment by Lycanzito** [comment]
> Solved on the [67433c0](https://github.com/opentibiabr/otservbr-global/commit/67433c0c1b4f62f0a87ca4d99723ad8d49747fa2)
> 
> Thanks.

=/
ok;
